### DISCUSSION TITLE
Page Numbers For Blog Posts

### DISCUSSION THREAD

**Discussion by the4rchangel** [daattali/beautiful-jekyll#774] [Q&A]
Is there a way to designate the page format to annotate pages to move between (i.e. links at the bottom of the blog page with 1,2,3) as opposed to the current "Older Posts" and "Newer Posts"? Thanks!

**Comment by daattali** [comment]
There's no native support for that. If you want to add page numbers you'll
need to overwrite the post layout, meaning you'll need to create a
_layouts/post.html file that looks similar to the existing one. In it
you'll see where the text "Next/Previous Posts" is defined. Instead of that
text you can try to use jekyll variables, see here
https://jekyllrb.com/docs/pagination/
